backend/src/zango/cli/start_project.py

```diff
@@ -98,7 +98,8 @@ def create_project(
     project_template_path = os.path.join(
         os.path.dirname(zango.cli.__file__), "project_template"
     )
-    command = f"{command} --template '{str(project_template_path)}'"
+    # command = f"{command} --template '{str(project_template_path)}'"
+    command = f'{command} --template "{project_template_path}"'
```

Comment:
Hi @nandinigthub , can you please remove this comment?
```suggestion
    command = f'{command} --template "{project_template_path}"'
```